package.json

```diff
@@ -1,39 +1,43 @@
 {
-  "private": true,
-  "scripts": {
-    "dev": "npm run development",
-    "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch-poll": "npm run watch -- --watch-poll",
-    "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "prod": "npm run production",
-    "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js"
-  },
-  "devDependencies": {
-    "bootstrap": "4.6.0",
-    "cross-env": "5.2.0",
-    "laravel-mix": "5.0.9",
-    "lodash": "4.17.20",
-    "resolve-url-loader": "3.1.2",
-    "sass": "1.32.5",
-    "sass-loader": "7.3.1"
-  },
-  "dependencies": {
-    "@bugsnag/js": "7.6.0",
-    "@bugsnag/plugin-vue": "7.6.0",
-    "axios": "0.21.1",
-    "chartist": "0.11.4",
-    "chartist-plugin-legend": "0.6.2",
-    "chartist-plugin-pointlabels": "0.0.6",
-    "jquery": "3.5.1",
-    "moment": "2.29.1",
-    "popper.js": "1.16.1",
-    "sweetalert2": "8.19.0",
-    "vue": "2.6.12",
-    "vue-flatpickr-component": "8.1.6",
-    "vue-moment": "4.1.0",
-    "vue-select": "3.11.2",
-    "vue-template-compiler": "2.6.12",
-    "vuelidate": "0.7.6"
-  }
-}
+    "private": true,
+    "scripts": {
+        "dev": "npm run development",
+        "development": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "watch": "cross-env NODE_ENV=development node_modules/webpack/bin/webpack.js --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "watch-poll": "npm run watch -- --watch-poll",
+        "hot": "cross-env NODE_ENV=development node_modules/webpack-dev-server/bin/webpack-dev-server.js --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "prod": "npm run production",
+        "production": "cross-env NODE_ENV=production node_modules/webpack/bin/webpack.js --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+        "build-apiary-nova-data-export": "cd nova-components/ApiaryNovaDataExport && npm run dev",
+        "build-apiary-nova-data-export-prod": "cd nova-components/ApiaryNovaDataExport && npm run prod",
+        "build-data-export": "cd nova-components/DataExport && npm run dev",
+        "build-data-export-prod": "cd nova-components/DataExport && npm run prod"
```

Comment:
What are these?

Reply:
Failed attempts that have now been removed.